_docs/examples/android.md

```diff
@@ -53,7 +53,7 @@ script.load()
 sys.stdin.read()
 {% endhighlight %}
 
-Note we use `this.m.value = 0` instead of `this.m = 0` to set the field's value. If there is also a method in this class called `m`, we need to use `this._m.value = 0` to set the value of field `m`.
+Note we use `this.m.value = 0` instead of `this.m = 0` to set the field's value. If there is also a method in this class called `m`, we need to use `this._m.value = 0` to set the value of field `m`.  In general, when looking at the properties of objects it will be necessary to use `.value` to access the objects those properties refer to.
```

Comment:
Just two tiny nitpicks here:
- `to access the objects` => `to access the values`, as fields may also be primitive values and not just objects
- `properties` => `fields`